### ISSUE TITLE
Sign errors in  amuse/ext/static_potentials.py

### ISSUE THREAD

**Issue by eerovaher** [amusecode/amuse#317]
**Describe the bug**
The method `get_gravity_at_point()` in the class `Abstract_Potential` returns the gradient of the gravitational potential, but should return the opposite value. Additionally, the method `flattened_potential()` wrongly returns a positive value for positive masses while e.g. `point_mass_potential()` correctly returns a negative 
value.

**To Reproduce**
```
from amuse.lab import *
from amuse.couple import bridge
from amuse.ext.static_potentials import Abstract_Potential
from matplotlib import pyplot as plt
import numpy as np

class Pointmass(Abstract_Potential):

    def get_potential_at_point(self, eps, x, y, z):
        return -constants.G*(1.|units.MSun)/(x**2+y**2)**0.5

Earth = Particles(1)
Earth.position = (1,0,0)|units.AU
Earth.velocity = (0,30,0)|units.km/units.s
Earth.mass = 6e24|units.kg
converter = nbody_system.nbody_to_si(Earth.x[0], Earth.mass[0])
gravity = BHTree(converter)
gravity.particles.add_particles(Earth)
Sun = Pointmass()
bridge = bridge.Bridge(use_threading=False)
bridge.add_system(gravity, (Sun,))
x = np.zeros(365)
y = np.zeros(365)
for t in range(365):
    bridge.evolve_model(t|units.day)
    x[t] = gravity.particles[0].x.value_in(units.AU)
    y[t] = gravity.particles[0].y.value_in(units.AU)
plt.plot(x,y)
plt.xlabel('x [AU]')
plt.ylabel('y [AU]')
plt.savefig('bug_demonstration.png')
plt.close()
bridge.stop()
gravity.stop()
``` 

**Expected behavior**
Physical potential values should be negative and gravitational acceleration should be the negative of the potential's gradient.

**Comment by ipelupessy** [comment]
thanks, yes, I think you are rightm the potentials in this file seems to be inconsistent..

**Comment by ipelupessy** [comment]
see #325, do you agree with these?

**Comment by ipelupessy** [comment]
mayby @vdhelm can comment?

**Comment by vdhelm** [comment]
my apologies, thanks for catching that, I could certainly have made those mistakes as I wrote this code with a single simulation in mind. I would guess that both this code and the code that uses this should both have a minus sign that I neglected, which would of course negate each other so my tests all worked fine. Feel free to correct this in a pull request and I'll adjust my code to match it.

**Comment by MJCWilhelm** [comment]
This appears to have been fixed in #325 , so I think this can be closed.
